### PR TITLE
[zh-cn]: update translation of `HTMLFormElement.elements` property

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/api/htmlformelement/elements/index.md [hunk 1]
```diff
@@ -1,29 +1,100 @@
...
+
+一个 {{domxref("HTMLFormControlsCollection")}}，其中包含表单中所有非图像控件。这是一个实时集合；如果向表单添加或删除表单控件，此集合将更新并更改。
+
+返回集合中的表单控件遵循树的前序深度优先遍历顺序出现在表单中。这称为**树顺序**。
+
+只返回以下元素：
+
+- {{HTMLElement("button")}}
+- {{HTMLElement("fieldset")}}
+- {{HTMLElement("input")}}（由于历史原因，为 [`type`](/zh-CN/docs/Web/HTML/Element/input#type) 类型的 `"image"` 元素都将被省略）。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/htmlformelement/elements/index.md:30]:
```suggestion
- {{HTMLElement("input")}}（由于历史原因，[`type`](/zh-CN/docs/Web/HTML/Element/input#type) 类型为 `"image"` 的元素都将被省略）。
```

File: files/zh-cn/web/api/htmlformelement/elements/index.md [hunk 2]
```diff
@@ -1,29 +1,100 @@
...
 slug: Web/API/HTMLFormElement/elements
+l10n:
+  sourceCommit: 1d6353ce109e9d94a7b5c7b6c40713d626d3628a
 ---
 
-小结
+{{APIRef("HTML DOM")}}
 
-**elements** 返回一个 {{domxref("HTMLFormControlsCollection")}} ([`HTMLCollection`](https://www.w3.org/TR/DOM-Level-2-HTML/html.html#ID-75708506)) 其中包含 FORM 的所有控件。需要注意的是，其中不包括 `type` 等于 `image` 的 [`input`](/zh-CN/docs/HTML/Element/Input) 元素。
+{{domxref("HTMLFormElement")}} 属性 **`elements`** 返回一个 {{domxref("HTMLFormControlsCollection")}}，其中列出了 {{HTMLElement("form")}} 元素中包含的所有表单控件。
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlformelement/elements/index.md:10]:
```suggestion
{{domxref("HTMLFormElement")}} 的 **`elements`** 属性返回一个 {{domxref("HTMLFormControlsCollection")}}，其中列出了 {{HTMLElement("form")}} 元素中包含的所有表单控件。
```

File: files/zh-cn/web/api/htmlformelement/elements/index.md [hunk 3]
```diff
@@ -1,29 +1,100 @@
...
+    用户名：
+    <input type="text" name="username" />
+  </label>
+  <label>
+    全名：
+    <input type="text" name="full-name" />
+  </label>
+  <label>
+    密码：
+    <input type="password" name="password" />
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlformelement/elements/index.md:55]: no spacing

```suggestion
    用户名：<input type="text" name="username" />
  </label>
  <label>
    全名：<input type="text" name="full-name" />
  </label>
  <label>
    密码：<input type="password" name="password" />
```

File: files/zh-cn/web/api/htmlformelement/elements/index.md [hunk 4]
```diff
@@ -1,29 +1,100 @@
...
+```js
+const inputs = document.getElementById("my-form").elements;
+const inputByIndex = inputs[0];
+const inputByName = inputs["username"];
 ```
 
-## 实例
+### 访问表单控件
+
+此示例获取表单的元素列表，然后遍历该列表，查找类型为 [`"text"`](/zh-CN/docs/Web/HTML/Element/input/text) 的 {{HTMLElement("input")}} 元素，以便对它们执行某种行为的处理。
```
Comment (yin1999) [inline files/zh-cn/web/api/htmlformelement/elements/index.md:68]:
```suggestion
此示例获取表单的元素列表，然后遍历该列表，查找类型为 [`"text"`](/zh-CN/docs/Web/HTML/Element/input/text) 的 {{HTMLElement("input")}} 元素，以便对它们执行某种形式的处理。
```

**Review by yin1999 — COMMENTED** [review]
nits:

**Review by yin1999 — APPROVED** [review]
:+1:

**Pull request merged**